lib/iris/tests/unit/coords/test_AuxCoord.py

```diff
@@ -639,5 +640,86 @@ def test_nanbounds_eq_self(self):
         self.assertEqual(co1, co1)
 
 
+class Test_masked__creation(tests.IrisTest, CoordTestMixin):
```

Comment:
I think strictly, all this should go in the _existing_ `Test__init__`, at the top of the file.
That is the 'standard' place, and they would be easier to find.
Though if so, a comment introducing these ones + separating them off would also be handy.

But, did you actively choose _not_ to do that, for other purposes ??

Reply:
@pp-mo I did actively choose not to include it in the `Test__init__` class, simply because it was dealing with masking specifically... but I don't want to split hairs here, so I'm totally happy to combine `Test_masked__creation` into `Test__init__`